Index: src/test/pages/ToDo2.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>public class ToDo2 {\n\n    //Teste\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/test/pages/ToDo2.java b/src/test/pages/ToDo2.java
--- a/src/test/pages/ToDo2.java	(revision f2bd80d066edde328781e968c51e65cac909f39d)
+++ b/src/test/pages/ToDo2.java	(date 1671283698749)
@@ -1,4 +1,2 @@
 public class ToDo2 {
-
-    //Teste
 }
